corehq/apps/reports/templates/reports/filters/bootstrap5/single_option.html

```diff
@@ -2,22 +2,32 @@
 {% load hq_shared_tags %}
 {% block filter_content %}
   {% if pagination.enabled %}
-    <select id="{{ css_id }}"
-            class="input form-control report-filter-single-option-paginated"
-            data-handler="{{ pagination.handler }}"
-            data-url="{{ pagination.url }}"
-            data-action="{{ pagination.action }}"
-            style="width: 100%"  {# todo B5: inline style #}
-            data-placeholder="{{ select.default_text }}"
-            data-selected="{{ select.selected|default:'' }}"
-            name="{{ slug }}"></select>
+    <select
+      id="{{ css_id }}"
+      class="input form-control report-filter-single-option-paginated w-100"
+      name="{{ slug }}"
+      data-handler="{{ pagination.handler }}"
+      data-url="{{ pagination.url }}"
+      data-action="{{ pagination.action }}"
+      data-placeholder="{{ select.default_text }}"
+      data-selected="{{ select.selected|default:'' }}"
+    ></select>
   {% else %}
-    <select id="{{ css_id }}"
-            data-select-options="{% html_attr select.options %}"
-            data-selected="{{ select.selected|default:'' }}"
-            class="report-filter-single-option"
-            style="width: 100%"  {# todo B5: inline style #}
-            data-bind="options: select_params, optionsText: 'text', optionsValue: 'val', {% if select.default_text %}optionsCaption: '{{ select.default_text|escapejs }}', {% endif %}value: current_selection"
-            name="{{ slug }}"></select>
+    <select
+      id="{{ css_id }}"
+      class="report-filter-single-option w-100"
```

Comment:
I see the old one didn't have `form-control`, but do you know why? Does `report-filter-single-option` make that irrelevant?

Reply:
it probably matters, but also there is the select 2 styling that overrides these anyway...
fixed here https://github.com/dimagi/commcare-hq/pull/35704/commits/6fd62f71fcbd2fc56936b57f3388471d4498df88